subsys/net/lib/dns/resolve.c

```diff
@@ -75,6 +75,7 @@ NET_BUF_POOL_DEFINE(dns_qname_pool, DNS_RESOLVER_BUF_CTR, DNS_MAX_NAME_LEN,
 		    0, NULL);
```

Comment:
Typo in the commit message `net/dns: Initialize default DNS context object`
s/dns_resovle_/dns_resolve/